src/ape/api/projects.py

```diff
@@ -127,6 +127,11 @@ def _create_compilers_list(
     ):
         compiler_list: List[Compiler] = []
         for key, compiler in cls.compiler_manager.registered_compilers.items():
+            contractTypes = [
+                str(x)
+                for x in contract_types.keys()
+                if key.strip(".") in contract_types[x].source_id
+            ]
```

Comment:
this seems to be working!

Reply:
What would happen if I had a contract named `sol.vy` ?

Reply:
oh i see that would not work. I could split it by `.` and use the first index though!

Reply:
this should be fixed in newest commit, and fixed the variable name to not be camel case